packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -88,8 +91,14 @@ class ExpansionPanel {
 }
 
 /// An expansion panel that allows for radio-like functionality.
+/// This means that at any given time, only one [ExpansionPanelRadio]
+/// can remain expanded at any given time.
```

Comment:
"at any given time" is repeated twice here - maybe just `At any given time, only one [ExpansionPanelRadio] can remain expanded.`?